src/studio/src/designer/backend.Tests/Factories/ModelFactory/XsdToJsonSchemaTests.cs

```diff
@@ -36,36 +37,29 @@ public void AsJsonSchema_ConvertXsdToJsonSchema_CorrectNumberOfPropertiesAndDefi
         [Fact]
         public void ConvertXsdToJsonSchemaAndBack_CorrectNumberOfPropertiesAndDefinitions()
         {
-            // Arrange
-            XmlReader xsdReader = XmlReader.Create(LoadTestData("Designer.Tests._TestData.xsd.melding-1603-12392.xsd"));
-            XsdToJsonSchema target = new XsdToJsonSchema(xsdReader);
-
-            // Act
-            JsonSchema actual = target.AsJsonSchema();
-
-            JsonSchemaToXsd jsonSchemaToXsd = new JsonSchemaToXsd();
+                // Arrange
+                XmlReader xsdReader = XmlReader.Create(LoadTestData("Model/xsd/melding-1603-12392.xsd"));
+                XsdToJsonSchema target = new XsdToJsonSchema(xsdReader);
+                xsdReader.Close();
 
-            XmlSchema xmlschema = jsonSchemaToXsd.CreateXsd(actual);
+                // Act
+                JsonSchema actual = target.AsJsonSchema();
 
-            FileStream file = new FileStream("Designer.Tests._TestData.xsd.melding-1603-12392b.xsd", FileMode.Create, FileAccess.ReadWrite);
-            XmlTextWriter xwriter = new XmlTextWriter(file, new UTF8Encoding());
-            xwriter.Formatting = Formatting.Indented;
-            xmlschema.Write(xwriter);
-
-            // Assert
-            Assert.NotNull(actual);
-            Assert.Equal(12, actual.Properties().Count);
-            Assert.Equal(19, actual.Definitions().Count);
+                // Assert
+                Assert.NotNull(actual);
+                Assert.Equal(12, actual.Properties().Count);
+                Assert.Equal(19, actual.Definitions().Count);                  
         }
-
+        
         private Stream LoadTestData(string resourceName)
         {
-            Assembly assembly = typeof(XsdToJsonSchemaTests).GetTypeInfo().Assembly;
-            Stream resource = assembly.GetManifestResourceStream(resourceName);
+            string unitTestFolder = Path.GetDirectoryName(new Uri(typeof(XsdToJsonSchemaTests).Assembly.CodeBase).LocalPath);
+            unitTestFolder = Path.Combine(unitTestFolder, @"..\..\..\_TestData\");
```

Comment:
With this change we can remove the two first item groups in the project file. No need to embed the resources if we don't use them in that way:
```
<ItemGroup>
    <Content Remove="_TestData\**\*.*" />
</ItemGroup>
<ItemGroup>
    <EmbeddedResource Include="_TestData\**\*.*" />
</ItemGroup>
```

Reply:
I only refactored the failing tests. So some tests are still using the embedded resource.. 
Gave refactoring the working tests as well, but new issues occur, so keeping the current structure.